scrapers/CopyToGallery.py

```diff
@@ -0,0 +1,151 @@
+import json
+import sys
+import os
+import requests
+
+import py_common.graphql as graphql
+import py_common.log as log
+
+SERVER_IP = "http://localhost:9999"
+# API key (Settings > Configuration > Authentication)
+STASH_API = ""
+SERVER_URL = SERVER_IP + "/graphql"
+
+find_gallery = False
+
+def call_graphql(query, variables=None):
+    headers = {
+        "Accept-Encoding": "gzip, deflate, br",
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "Connection": "keep-alive",
+        "DNT": "1",
+        "ApiKey": STASH_API
+    }
+    json = {'query': query}
+    if variables is not None:
+        json['variables'] = variables
+    try:
+        response = requests.post(SERVER_URL, json=json, headers=headers)
+        if response.status_code == 200:
+            result = response.json()
+            if result.get("error"):
+                for error in result["error"]["errors"]:
+                    raise Exception("GraphQL error: {}".format(error))
+            if result.get("data"):
+                return result.get("data")
+        elif response.status_code == 401:
+            log.debug("401 returned from callGraphQL")
+            # debug("[ERROR][GraphQL] HTTP Error 401, Unauthorised.")
+            return None
+        else:
+            raise ConnectionError("GraphQL query failed:{} - {}".format(response.status_code, response.content))
+    except Exception as err:
+        log.error(err)
+        # debug(err)
+        return None

```

Comment:
from a quick look the above can be replaced by
```python
find_gallery = False

def call_graphql(query, variables=None):
    return graphql.callGraphQL(query, variables)
```
Using the imported callGraphQL function makes the code cleaner and already includes the stash configuration from `config.py` in the `py_common` folder

Reply:
Perfect, thanks for bringing this to my attention.